terraform/modules/forest_node/user-data.sh

```diff
@@ -55,28 +59,73 @@ cat << EOF > "/home/${NEW_USER}/forest_data/config.toml"
 data_dir = "/home/${NEW_USER}/forest_data/data"
 EOF
 
+
+sudo --user="${NEW_USER}" -- docker network create forest
+
 # Run the Forest Docker container as the created user.
 sudo --user="${NEW_USER}" -- \
   docker run \
   --detach \
-  --name=forest \
+  --network=forest \
+  --name=forest-"${CHAIN}" \
   --volume=/home/"${NEW_USER}"/forest_data:/home/"${NEW_USER}"/data \
   --publish=1234:1234 \
+  --publish=6116:6116 \
   --restart=always \
   ghcr.io/chainsafe/forest:latest \
   --config=/home/"${NEW_USER}"/data/config.toml \
-  --auto-download-snapshot \
   --encrypt-keystore false \
-  --chain="${CHAIN}"
+  --auto-download-snapshot \
+  --chain="${CHAIN}" 
 
-# Run the Watchtower Docker container.
-# It monitors running Docker containers and watches for changes to the images that those containers were originally started from.
+# It monitors running Docker containers and watches for changes to the images that those containers were originally started from. 
 # If Watchtower detects that an image has changed, it will automatically restart the container using the new image.
+# Run the Watchtower Docker container as created user.
 sudo --user="${NEW_USER}" -- \
   docker run \
   --detach \
   --name=watchtower \
+  --network=forest \
   --volume=/var/run/docker.sock:/var/run/docker.sock \
   --restart=unless-stopped \
   containrrr/watchtower \
   --label-enable --include-stopped --revive-stopped --stop-timeout 120s --interval 600
+
+# Set-up  New Relic Agent For logs collection and Infrastruture Metrics
+sudo --user="${NEW_USER}" -- \
+  bash -c "curl -Ls https://download.newrelic.com/install/newrelic-cli/scripts/install.sh | bash && \
+  sudo NEW_RELIC_API_KEY=""${NEW_RELIC_API_KEY}"" \
+       NEW_RELIC_ACCOUNT_ID=""${NEW_RELIC_ACCOUNT_ID}"" \
+       NEW_RELIC_REGION=EU \
```

Comment:
Should the NR region be parametrised?

Reply:
I'd say this should be parametrised. If it is not, someone changing the node to NYC may be surprised he can't without changing a seemingly random shell script.